docs/tutorials/quickstart.md

```diff
@@ -1,8 +1,194 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.15.2
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
 # Quickstart
 
-```{note}
-This is a placeholder for a section in the new {ref}`jax-tutorials`.
+**JAX is array-oriented computation (*a la* [NumPy](https://numpy.org/)), with automatic differentiation and JIT compilation for high-performance machine learning research**.
+
+This document gives a quick overview of essential JAX features:
+
+* JAX provides a unified NumPy-like interface to computations that run on CPU, GPU, or TPU, in single-host or distributed settings.
+* JAX features built-in Just-In-Time compilation via [Open XLA](https://github.com/openxla), an open-source machine learning compiler ecosystem.
+* JAX programs support efficient evaluation of gradients via its automatic differentiation transformations.
+* JAX programs can be automatically vectorized to efficiently map custom kernels across batches.
+
+## Installation
+
+JAX can be installed for CPU on Linux, Windows, and OSX directly from the [Python Package Index](https://pypi.org/project/jax/):
+```
+pip install "jax[cpu]"
+```
+For more detailed installation information, including installation with GPU support, sett {ref}`installation`.
+
+## JAX as NumPy
+
+Most JAX usage is through the familiar {mod}`jax.numpy` API, which is typically imported under the `jnp` alias:
+
+```{code-cell}
+import jax.numpy as jnp
+```
+
+With this import, you can immediately use JAX in a similar manner to typical NumPy programs,
+including using NumPy-style array creation functions, Python functions and operators, and
+array attributes and methods:
+
+```{code-cell}
+def selu(x, alpha=1.67, lmbda=1.05):
+  return lmbda * jnp.where(x > 0, x, alpha * jnp.exp(x) - alpha)
+
+x = jnp.arange(5.0)
+print(selu(x))
+```
+
+You'll find a few differences between JAX arrays and NumPy arrays once you begin digging-in;
+these are explored in  [🔪 JAX - The Sharp Bits 🔪](https://jax.readthedocs.io/en/latest/notebooks/Common_Gotchas_in_JAX.html).
+
+## Using {func}`~jax.jit` to speed up functions
```

Comment:
Maybe in this section's header, or maybe just at the end of its body, we should call out automatic parallelization over multiple devices. We don't want to bury the lede too much!